modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/configuration/permission/propagation/DLFolderPortletConfigurationPermissionPropagation.java

```diff
@@ -134,11 +150,102 @@ private long _getClassPK(PortletRequest portletRequest) {
 		return resourcePrimKeys[0];
 	}
 
+	private int _getFileEntriesCount(
```

Comment:
Should this method be named _getFileEntryCountInDirectory, that make its purpose clearer.

Reply:
I put the `folderId` as a parameter, so it should be descriptive of the function's purpose, which is to get file entries in the folder. Thanks.